src/tracing/json/json_tracing.h

```diff
@@ -52,6 +52,7 @@ class JsonBackend : public ::chip::Tracing::Backend
     void TraceEnd(const char * label, const char * group) override;
     void TraceInstant(const char * label, const char * group) override;
     void TraceCounter(const char * label) override;
+    void TraceMetric(const char * label, int val) override;
```

Comment:
int size is platform-specific. Should we int32_t or int64_t instead?

Reply:
I wonder if we should name things `TRACE_METRIC_INT / TraceMetricInt` to specify the type. I am unsure  if we will ever want different types of values.

Do you happen to have a list of where this would be used besides wifi RSSI?